Cargo.toml

```diff
@@ -40,7 +40,7 @@ trybuild = "=1.0.67"
 
 # For test on MSRV.
 once_cell = "=1.14"
-serde = "=1.0.156"
+serde = "=1.0.159"
 toml = "=0.5.9"
```

Comment:
I think these lines (and line 39's trybuild's `=` requirement) can be removed because MSRV has increased.